app/scripts/modules/titus/src/instance/details/instance.details.controller.js

```diff
@@ -122,17 +122,7 @@ module(TITUS_INSTANCE_DETAILS_INSTANCE_DETAILS_CONTROLLER, [
             if (overrides.instanceDetailsLoaded) {
               overrides.instanceDetailsLoaded();
             }
-
-            // Network interfaces are needed for any IPv6 addresses.
-            return InstanceReader.getInstanceDetails(accountDetails.environment, region, instanceDetails.agentId);
-          })
-          .then(instance => {
-            if (instance && instance.networkInterfaces) {
-              $scope.instance.ipv6Addresses = flatMap(instance.networkInterfaces, i =>
-                i.ipv6Addresses.map(a => a.ipv6Address),
-              );
-            }
-          }, autoClose);
+          });
```

Comment:
`autoClose` is still needed here: `.then( () => {}, autoClose)`

Reply:
thanks , fixed it